src/chart/funnel/FunnelSeries.ts

```diff
@@ -94,7 +94,15 @@ export interface FunnelSeriesOption
 
     funnelAlign?: HorizontalAlign | VerticalAlign
 
+    exitShape?: 'rect' | 'none'
+
     data?: (OptionDataValueNumeric | OptionDataValueNumeric[] | FunnelDataItemOption)[]
+
+    dynamicHeight?: boolean
+
+    exitWidth?: string
```

Comment:
Since we provide `exitWidth`, the user can set `exitWidth: '100%'` to make the exit shape to be rect, why do we need an extra option `exitShape`?

Reply:
Because when dynamicHeight is  true, set exitWidth to make whole funnel a trapezoidal. When exitWidth is '100%', it's mean whole funnel a rect, it's unreasonable . And when showRate is true, and not in dynamicHeight, set exitWidth to make exit piece a trapezoidal. Set exitShape rect when dynamicHeight is true and default situation, it will make exit piece a rect, beacaue exitWidth can't control exit piece in this two situation.

Reply:
I read the description again and I found that I misunderstood the concept of `exitWidth` completely. I think this is a useful feature but a better name is required to let the developers understand this is for the overall funnel rather than the last one because the *exist* suggest it to be the last one.

Reply:
Yes, I agree with you, my mistake, and I will correct it